src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -108,9 +112,11 @@ export class NamespaceStore extends KubeObjectStore<Namespace> {
     return namespaces;
   }
 
-  getSubscribeApis() {
+  async getSubscribeApis() {
```

Comment:
Why it's became `async`?

Reply:
Because we need to wait for `contextReady`